api-auth/service/README.md

```diff
@@ -0,0 +1,131 @@
+# Service
+
+A service represents an application and defines a set of public and protected urls
+and the route to access to them.
+
+The expected format for each service file is:
+
+```javascript
+{
+  // service name (unique among rest of services)
+  "name": "service-name",
+
+  // internal host (behind kong)
+  "host": "http://my-service:8888",
+
+  // list of urls protected by "kong-oidc-auth" plugin
+  // all of these urls provide the X-Outh-Token header
+  // or are redirected to keycloak to authenticate
+  "oidc_endpoints": [
+    {
+      // endpoint name (unique among rest of OIDC endpoints in this service)
+      "name": "protected",
+
+      // internal url
+      "url": "/protect-me-please",
+
+      // [optional] external url, defaults to "/{realm}/{name}{url}" where
```

Comment:
This is a little confusing: I guess that `url` refers to the previous setting, which you refer to in the comment as "internal url"? but that `name` doesn't refer to the "endpoint name" above? or am I totally wrong?

Reply:
These conventions are from Gateway Manager. We should address them over there on a breaking change and then propagate them back: https://github.com/eHealthAfrica/gateway-manager